### PR TITLE
TRUNK-5564:PersonAddressValidator - Throws exception instead of logging

### PR DESCRIPTION
**Pull request by jwnasambu**
## Description of what I changed
I coded the PersonAddressValidator  in a way that avoids throwing exceptions.

## Checklist: I completed these to help reviewers :)
https://issues.openmrs.org/browse/TRUNK-5564

### CODE REVIEW

**Comment by jwnasambu** [comment]
@dkayiwa I have made some changes kindly feel free to review my work at your convenient time.

**Comment by dkayiwa** [comment]
@jwnasambu are you still working on this?

**Comment by jwnasambu** [comment]
Yes am still working on it

    On Thursday, November 28, 2019, 9:54:26 AM GMT+2, dkayiwa <notifications@github.com> wrote:  
 
 
@jwnasambu are you still working on this?

**Comment by jwnasambu** [comment]
@dkayiwa I have pushed some changes feel free to advice and commend.

File: api/src/main/java/org/openmrs/validator/PersonAddressValidator.java [hunk 1]
```diff
@@ -62,13 +62,10 @@ public void validate(Object object, Errors errors) {
...
 			log.debug(this.getClass().getName() + ".validate...");
 		}
-		
-		if (object == null) {
-			throw new IllegalArgumentException("The personAddress object should not be null");
-		}
-		
 		PersonAddress personAddress = (PersonAddress) object;
-		
+		if (personAddress == null) {
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/validator/PersonAddressValidator.java]: Is there any reason why you moved this from where it was above to here?

**Comment by kangbreder** [comment]
@jwnasambu  Please add some changes to this PR

**Pull request closed without merging**